spec/features/consumer/account/cards_spec.rb

```diff
@@ -4,7 +4,8 @@
   include AuthenticationWorkflow
   describe "as a logged in user" do
     let(:user) { create(:user) }
-    let!(:card) { create(:credit_card, user_id: user.id, gateway_customer_profile_id: 'cus_AZNMJ') }
+    let!(:card) { create(:credit_card, user_id: user.id, gateway_customer_profile_id: 'cus_AZNMJ', is_default: true) }
+    let!(:card2) { create(:credit_card, user_id: user.id, gateway_customer_profile_id: 'cus_FDTG') }
```

Comment:
Only problem with this is that the spec tests the ability to switch the default card, so that by the end of the spec the `non_default_card` is in fact the default card... I will make the change anyway.

Reply:
Done